news/2025/2025-02-21-osumania-7k-world-cup-2025-finals-recap.md

```diff
@@ -0,0 +1,217 @@
+
+At a first glance, the Philippines seemed to have an advantage in the tiebreaker, as EX3 maps are the most similar maps to actual tiebreakers. Surprisingly, Japan's performance far surpassed our initial expectations, maintaining the lead for a significant portion of the tiebreaker's runtime. The most critical moment came in the final section of the chart, where the Philippines managed to close the gap by a hair's breadth. Just when it seemed like Japan would pull ahead with a 5,000-point lead, the Philippines made a stunning comeback during the last long note section, securing victory by a mere **101 points** — two points short of a two-digit number! This shocking turn of events left both the audience and commentators in disbelief.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2025-02-14-watch-this.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+This was without a doubt one of the most intense and hyped up matches in the history of the osu!mania 7K World Cup. Both teams fought fiercely until the very bitter end, with the outcome remaining mostly unpredictable and full of unexpected upsets.
+
+Congratulations to the Philippines for advancing to the Grand Finals, and a huge round of applause to Japan for performing as well as they did on a match that will be found in the annals of osu!mania tournament history! Let's see how stronger Japan will become in the next 7K World Cups to come.
+
+## Interview
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-no-group {
+        color: #FFFFFF;
+    }
+</style>
+
+![](/wiki/shared/news/2025-02-21-osumania-7k-world-cup-2025-finals-recap/tyrcs.png)
+
+[tyrcs](https://osu.ppy.sh/users/13026904) needs little to no introduction. 7K player extraordinaire, their performance during the tournament has left many users in awe. Even those who were already familiar with their skills were still impressed by consistently providing us with amazing scores throughout the tournament. The scores highlights section may've as well been theirs!
```

Comment:
the score highlight section doesn't even mention tyrcs this week...

```suggestion
[tyrcs](https://osu.ppy.sh/users/13026904) needs little to no introduction. 7K player extraordinaire, their performance during the tournament has left many users in awe. Even those who were already familiar with their skills were still impressed by consistently providing us with amazing scores throughout the tournament.
```

Reply:
While it doesn't this week, they were very present throughout the tournament in these highlights. This is making reference to that rather than this week in particular.